packages/react-instantsearch-native/.babelrc

```diff
@@ -48,7 +48,7 @@
             "targets": {
               "browsers": [
                 "last 2 versions",
-                "ie >= 10"
+                "ie >= 9"
```

Comment:
why this one 9 specific? I assume RN uses JSC, which has far above IE9 capabilities (even has modules IIRC)

Reply:
I will revert all the `.babelrc` to `ie >= 9` it's the value that we have today in the project. For React Native the impact of changing this to a lower version is very low because we mainly use the `core` package. This dependency is already transpile with a IE9 preset.